fvm/state/ledger.go

```diff
@@ -49,6 +58,26 @@ func (m MapLedger) Delete(owner, controller, key string) {
 	delete(m.Registers, fullKey(owner, controller, key))
 }
 
+func (m MapLedger) RegisterUpdates() ([]flow.RegisterID, []flow.RegisterValue) {
+	data := make(flow.RegisterEntries, 0, len(m.Registers))
+
+	for _, v := range m.Registers {
+		data = append(data, v)
+	}
+
+	sort.Sort(&data)
```

Comment:
Why sort them? Is this needed?

Reply:
No. This was copied from delta (where it is needed because of determinism). Since MapLedger is only used in tests, this isn't needed. I'll remove it.